### PR TITLE
added no conversations image, added contacts screen

### PR DESCRIPTION
**Pull request by Prophetaa**

### CODE REVIEW

**Comment by Prophetaa** [comment]
Screen Shot 2019-07-18 at 01 55 26 AM
Screen Shot 2019-07-18 at 01 55 32 AM
Screen Shot 2019-07-18 at 01 55 38 AM

File: components/ListItem.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+import { View, Text, Image, TouchableHighlight } from 'react-native';
+import { StyleSheet } from 'react-native';
+
+const ListItem = ({ imageSource, text, onPress }) => {
+	return (
+		<TouchableHighlight onPress={onPress}>
+			<View style={styles.container}>
+				<Image
+					style={styles.image}
+					source={require('../assets/plus.png')}
```
Comment (frogThanos) [inline components/ListItem.js]: I would move assets like images, colors, fonts, strings, ... etc to a config folder. Inside the folder, you can create an index.js file and export objects that contain all that info to the rest of the app, that way when you want to use/find an image (and you have 200 of them) you can just look it up in this folder. This works well especially if you use the same image more than once. Check any of our react, react-native projects to see more 😛

File: .expo-shared/assets.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "72bc3af7ca1cb5507f3ff16ccb38dfeb378f194661391ccd8a544b707cc76836": true,
```
Comment (frogThanos) [inline .expo-shared/assets.json]: what kind of assets are these?

File: helpers/AppNavigation.js [hunk 3]
```diff
@@ -2,6 +2,8 @@ import { createStackNavigator, createAppContainer } from 'react-navigation';
 import HomeScreen from '../screens/HomeScreen';
 import LoginScreen from '../screens/LoginScreen';
 import SplashScreen from '../screens/SplashScreen';
+import ContactsScreenStyles from '../styling/ContactsScreen.styles';
```
Comment (frogThanos) [inline helpers/AppNavigation.js]: I would move the styles into a file next to the component like so: 
Screenshot 2019-07-18 at 09 31 33

That way you don't have to navigate to the other side of the project just to find the component's styles. Also, keep in mind that this falls under the umbrella of writing your components in a modular way.

File: redux/actions/conversations.js [hunk 4]
```diff
@@ -7,6 +7,7 @@ export const FETCHING_CONVERSATIONS = 'FETCHING_CONVERSATIONS';
 export const fetchMyConversations = () => async (dispatch, getState) => {
```
Comment (frogThanos) [inline redux/actions/conversations.js]: This file is small for now, you should consider splitting up your types and actions in order to accommodate for scalability.

File: screens/ContactsScreen.js [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+import { List, Modal, Provider, Button } from '@ant-design/react-native';
+import PhoneInput from 'react-native-phone-input';
+import styles from '../styling/ContactsScreen.styles';
+import ListItem from '../components/ListItem';
+
+const Item = List.Item;
+
+class ContactsScreen extends Component {
+	async componentDidMount() {}
+	state = { phoneNumber: null };
```
Comment (frogThanos) [inline screens/ContactsScreen.js]: Setting your components default state should not happen in componentDidMount. If you want to change your state use ` setState()` in componentDidMount. Default state should be declared in the constructor for class components and ` useState(0)` for functional components. You can google it for more info about why. 😛

File: screens/LoginScreen.js [hunk 6]
```diff
@@ -13,8 +13,8 @@ import {
 import { signUp } from '../redux/actions/user';
 
 class LoginScreen extends Component {
-	componentWillReceiveProps() {
-		if (this.props.currentUser) {
+	componentWillUpdate(newProps) {
+		if (newProps.currentUser) {
```
Comment (frogThanos) [inline screens/LoginScreen.js]: Can you explain to me what you are checking for here? if the user is logged in? Why not just do it in componentDidMount?

File: helpers/AppNavigation.js (hunk 3 above)
Comment (Prophetaa) [reply helpers/AppNavigation.js]: Will do! Thank you

**Pull request merged**